eloquent/distribution.yaml

```diff
@@ -437,7 +437,7 @@ repositories:
       tags:
         release: release/eloquent/{package}/{version}
       url: https://github.com/ros2-gbp/cyclonedds-release.git
-      version: 0.5.1-2
+      version: 0.7.0-2
```

Comment:
We should probably also update the source branch below too.

Reply:
Good point - updated. Thanks.